src/timesync/timesyncd-server.c

```diff
@@ -60,6 +60,14 @@ ServerAddress* server_address_free(ServerAddress *a) {
         return mfree(a);
 }
 
+static void enable_ntp_server_defer_event(Manager *m, ServerType type) {
+        int r = sd_event_source_set_enabled(m->deferred_ntp_server_event_source, SD_EVENT_ONESHOT);
+                if (r < 0)
+                        log_debug_errno(r, "Failed to reenable system ntp server change event source!");
+        m->ntp_server_change_mask |= 1U << type;
```

Comment:
i presume you want to call bus_manager_emit_ntp_server_changed() here too?

Reply:
yes, added it.